tools/render-test/bind-location.cpp

```diff
@@ -328,6 +328,23 @@ void BindSet::calcValueLocations(const BindLocation& location, Slang::List<BindL
     }
 }
 
+// Finds the first category from layout reflection that represents an actual value
+// i.e. that is not ExistentialType or ExistentialObject.
+template<typename LayoutReflectionType>
+slang::ParameterCategory findOrdinaryValueCategory(LayoutReflectionType* layout)
```

Comment:
Right now, the layout/binding logic tries to use "ordinary" as a synonym for `LayoutResourceKind::Uniform` (just because "uniform" can imply other things, and we need a term to refer to "just bytes"). This function is using "ordinary" to mean something different, and maybe we need a better term.

How about `firstNonExistentialValueCategory` since that would describe exactly what is going on?